src/unabto/unabto_config_defaults.h

```diff
@@ -109,6 +109,11 @@
 #define NABTO_ENABLE_CONNECTION_ESTABLISHMENT_ACL_CHECK 0
 #endif
 
+/** Query at tunnel open request whether a client is allowed access */
```

Comment:
Lets remove this and just force people to implement the required function in the tunnel